composer.json

```diff
@@ -29,25 +28,21 @@
             "homepage": "https://atymic.dev"
         }
     ],
+    "homepage": "https://socialiteproviders.com",
+    "support": {
+        "issues": "https://github.com/socialiteproviders/manager/issues",
+        "source": "https://github.com/socialiteproviders/manager"
+    },
     "require": {
         "php": "^8.0",
         "illuminate/support": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0",
```

Comment:
May as well drop versions that dont support php 8 at least?

Reply:
Good idea, see https://github.com/SocialiteProviders/Manager/pull/222/commits/1bed9a39cc7d1ba9a020f29b74f69fbc990ca755